turtlebot3c-bringup-snap/snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: turtlebot3c-bringup
-adopt-info: turtlebot3c-bringup
+version: '0.1'
```

Comment:
Same question as for [humble](https://github.com/canonical/turtlebot3c-snap/pull/76#discussion_r1840910091)